source/deployment/desktop-app-deployment.rst

```diff
@@ -106,7 +108,10 @@ You can distribute the official Windows Desktop App silently to end users, pre-c
         echo }
       ) > %APPDATA%\Mattermost\config.json
 
+.. note::
+  Instead of using the ``rem`` command to install the Desktop App into a shared folder, you can also copy the executable to the folder before running it. This allows the shared folder to only require read-only permissions.
```

Comment:
I'm not sure why `rem` appear at this sentence because `rem` just makes a comment for batch file.

Reply:
@yuya-oc Thanks! Fixed